source/val/validate_ray_query.cpp

```diff
@@ -263,6 +263,17 @@ spv_result_t RayQueryPass(ValidationState_t& _, const Instruction* inst) {
       break;
     }
 
+    case spv::Op::OpRayQueryGetClusterIdNV: {
```

Comment:
Optionally, check that `Intersection` is a 32-bit integer constant.

Reply:
Updated